### PR TITLE
FreeBSD Graphics Team Q2 report

### PR DESCRIPTION
**Pull request by zeising**
Here is the FreeBSD Graphics Team Q2 report.

### CODE REVIEW

File: 2019q2/graphics.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+evdev was enabled in the default configuration late 2018.  Building on that,
+we have enabled IBM/Lenovo trackpoints, and elantech and synaptics touchpads by
+default as well.
+
+The input device library libinput has been updated, as the last in a series of
+updates bringing the userland input stack up to date.  This is work that was
+started in 2018.
+
+We have made several improvements to the drm kernel drivers.
+A longstanding memory leak in the Intel (i915) driver has been fixed, and
```
Comment (debdrup) [inline 2019q2/graphics.md]: s/longstanding/long\-standing/

File: 2019q2/graphics.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+pseudofs) works might prevent a fully functional debugfs and we might have to
+look into adding to pseudofs or use another framework.
+
+The new in-kernel drm driver for VirtualBox, `vboxvideo.ko` has been proted from
+Linux.
+Virtual console won't update after loading the driver, but X and Wayland
+based compositors are working.
+This is experimental and still a work in progress.
+
+
```
Comment (debdrup) [inline 2019q2/graphics.md]: Extraneous newline.

Comment (zeising) [reply 2019q2/graphics.md]: Fixed

File: 2019q2/graphics.md (hunk 1 above)
Comment (zeising) [reply 2019q2/graphics.md]: Fixed

**Comment by zeising** [comment]
Issues should be fixed now.  Please, if possible, hold on for a bit with merging. We have a graphics team meeting tonight, and I'd like to give the rest of the team one final opportunity to chime in. :)
I'll let you know here once again when we're OK.

**Comment by zeising** [comment]
All done, ready to go!
Thanks for waiting for us!

File: 2019q2/graphics.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Mesa has been updated to 18.3.2 and switched from using `devel/llvm60` to use
+the Ports default version of llvm, currently `devel/llvm80`.
+
+Several userland Xorg drivers, applications, and libraries have been updated,
+and other improvements to the various userland components that make up the
+Graphics Stack have been made.
+
+We have also continued our regularly scheduled bi-weekly meetings, although work
+remains in sending out timely meeting minuets afterwards.
```
Comment (debdrup) [inline 2019q2/graphics.md]: s/minuets/minutes/

**Pull request merged**